### PR TITLE
feat: ledger guide

### PR DESCRIPTION
**Pull request by obiyankenobi**
Guidelines: https://docs.google.com/document/d/1QI7DHd3HIyhKWydlmjBbB-dlUXquqAd-4f4ED_-wgAU/edit

Example: https://medium.com/rise-vision/how-to-use-ledger-nano-with-rise-ef781d2997fc

### CODE REVIEW

File: content/pages/ledger.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+date: 2020-02-27
+
+
+Hathor currently supports the [Ledger Nano S](https://www.ledger.com/products/ledger-nano-s) hardware wallet. Hardware
+wallets are dedicated external devices that store your private keys. By using hardware wallets, your funds are safe
+even when using an infected or untrusted PC.
+
+With Ledger, you can:  
+&nbsp; (i) see all your funds, for all your tokens;  
+&nbsp; (ii) send HTR transactions;
```
Comment (pedroferreira1) [inline content/pages/ledger.md]: Maybe add '(iii) get an address to receive funds'?

File: content/pages/ledger.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+3. On Ledger, you can scroll left/right to see the full address. If all is correct, press both buttons;
+![Confirm address](https://drive.google.com/thumbnail?id=1k8E3ZQaB3jvX2vd1VfgT9MrX9ckTBdpS&sz=w1024)
+
+### Sending HTR
+
+1. Go to the Send tokens menu on Hathor Wallet;
+2. Input the address and amount to be sent;
+3. Click the 'Send Tokens' button. A modal will show up asking to confirm the information on Ledger;
+![Confirm transaction modal](https://drive.google.com/thumbnail?id=1g9bZLG3tq4q-ycp3_73xhq61PLgcUO7Z&sz=w1024)
+4. Validate the information on your Ledger. On can scroll left/right to see the address and amount;
```
Comment (pedroferreira1) [inline content/pages/ledger.md]: Typo: You can scroll...

File: content/pages/ledger.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+title: Using Ledger Nano with Hathor
+date: 2020-02-27
+
+
+Hathor currently supports the [Ledger Nano S](https://www.ledger.com/products/ledger-nano-s) hardware wallet. Hardware
+wallets are dedicated external devices that store your private keys. By using hardware wallets, your funds are safe
+even when using an infected or untrusted PC.
+
+With Ledger, you can:  
+&nbsp; (i) see all your funds, for all your tokens;
```
Comment (msbrogli) [inline content/pages/ledger.md]: I'd say "check all your funds".

File: content/pages/ledger.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+With Ledger, you can:  
+&nbsp; (i) see all your funds, for all your tokens;  
+&nbsp; (ii) send HTR transactions;  
+
+Support for sending other tokens is still under development.
+
+## Before you begin
+
+. make sure your Ledger Nano S has been initialized;
```
Comment (msbrogli) [inline content/pages/ledger.md]: Initial letter as a capital letter.

File: content/pages/ledger.md (hunk 1 above)
Comment (obiyankenobi) [reply content/pages/ledger.md]: Done

File: content/pages/ledger.md (hunk 2 above)
Comment (obiyankenobi) [reply content/pages/ledger.md]: done

File: content/pages/ledger.md (hunk 3 above)
Comment (obiyankenobi) [reply content/pages/ledger.md]: done

File: content/pages/ledger.md (hunk 4 above)
Comment (obiyankenobi) [reply content/pages/ledger.md]: done

**Pull request merged**